docs/rules/filename-case.md

```diff
@@ -68,11 +68,19 @@ You can set the `cases` option to allow multiple cases:
 
 ### ignore
 
-Type: `RegExp[]`\
+Type: `Array<string | RegExp>`\
 Default: `[]`
 
 Filenames to ignore.
 
```

Comment:
Can you explicitly say that if a string is given, it's interpreted as a regex inside a string.